src/config_impl.hpp

```diff
@@ -53,9 +53,6 @@ std::vector<
 >
 pattern_opts;
 
-extern
-std::shared_ptr<rapidjson::Document> source;
```

Comment:
Currently, `handystats::config::pattern_opts` contains configuration for each of possible pattern. Configuration is in the form of `rapidjson::Value*` that is a pointer to this `source` subelement.